tests/pytests/unit/modules/test_salt_version.py

```diff
@@ -21,7 +21,7 @@ def test_mocked_objects():
     for k, v in salt.version.SaltStackVersion.LNAMES.items():
         assert k == k.lower()
         assert isinstance(v, tuple)
-        if sv.new_version(major=v[0]) and not sv.can_have_dot_zero(major=v[0]):
+        if sv.new_version(major=v[0]):
```

Comment:
So, from 3006 onward, the `.0` must always be present, the existing logic was correct, but SaltStackVersion is now reporting it differently?

Reply:
Ok, I see why this had to change.
And it looks like it hasn't broken other functionality.
```
❯ python
Python 3.10.2 (main, Nov 15 2022, 11:41:37) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import salt.version
>>> v = salt.version.SaltStackVersion.parse("3006")
>>> str(v)
'3006.0'
>>> v.info
v.info
>>> v.info
(3006, 0)
>>> v = salt.version.SaltStackVersion.parse("3005")
>>> str(v)
'3005'
>>> v.info
(3005,)
>>>
```